docs/tutorials/99_other/oemetadata.md

```diff
@@ -0,0 +1,96 @@
+# OEMetaData
+
+<!-- keep img below title and without align="left"  -->
+<img src="https://raw.githubusercontent.com/OpenEnergyPlatform/academy/develop/docs/data/img/OEP_logo_2_no_text.svg" alt="OpenEnergy Platform" height="75" width="75" />
+
+## Learnings
+This tutorial will enable you to:
+
+- know what OEMetaData are
+- create and modify OEMetaData
+
+## Introduction
+
+This tutorial will always refer specifically to OEMetaData (the Metadata used on the [Open Energy Platform](https://openenergyplatform.org/) (OEP)). For a more general overview please see the [course on Metadata](https://openenergyplatform.github.io/academy/courses/07_metadata/).
+
+## Important Information on OEMetaData
+
+- OEMetaData follow the specifications of one of the releas versions of the OEP. At the time v1.6.0, soon 2.0.0.
+- OEMetaData are always a JSON-file, which can refer to other elements, e.g. websides, classes from the [Oepen Energy Ontology](https://openenergyplatform.org/viewer/oeo/) (OEO) etc.. The OEMetaData in version v1.6.0 always describes one whole dataset to a table. 
+- OEMetaData follow a pattern. Find the key descriptions for the newest version of the OEMetaData on [GitHub](https://github.com/OpenEnergyPlatform/oemetadata/blob/develop/metadata/v160/metadata_key_description.md).
+- There can always only be one version of the OEMetaData dataset on the OEP! Meaning that the livetime of OEMetaData is critical because othervise there might be multiple datasets to one table. In order for that not to happen the published OEMetaData for a published table on the OEP stay like this until there is a new version of the OEMetaData dataset. (For more information on the publishing process see [this course](https://openenergyplatform.github.io/academy/courses/04_upload/).)  
```

Comment:
... new version of OEMetaData dataset ... : Hier meinte ich das eine neue version des datensatzes. OEMetaData Datensatz hört sich nach neuen metadaten an.

Reply:
Also lieber nur OEOMetaData ohne dataset?

Reply:
yes